user/reference/trusty.md

```diff
@@ -13,58 +13,35 @@ available in the Trusty environment.
+To use Ubuntu Trusty, add the following to your
 `.travis.yml`.
 
 ```yaml
 dist: trusty
-sudo: required
 ```
 {: data-file=".travis.yml"}
 
-Or to route to sudo-less:
-
-```yaml
-dist: trusty
-sudo: false
-```
-{: data-file=".travis.yml"}
-
-This is enabled for both public and private repositories.
 
 If you'd like to know more about the pros, cons, and current state of using
-Trusty, including details specific to container-based, read on ...
+Trusty, read on.
 
-## Fully-virtualized via `sudo: required`
-
-When specifying `sudo: required`, Travis CI runs each build in an isolated
-[Google Compute Engine](https://cloud.google.com/compute/docs/) virtual machine
-that offer a vanilla build environment for every build.
-
-This has the advantage that no state persists between builds, offering a clean
-slate and making sure that your tests run in an environment built from scratch.
+Travis CI runs each build in an isolated [Google Compute Engine](https://cloud.google.com/compute/docs/) virtual machine that offers a vanilla build environment for every build.
 
 Builds have access to a variety of services for data storage and messaging, and
 can install anything that's required for them to run.
 
-## Container-based with `sudo: false`
+## Linux infrastructure
+
+Travis CI runs each build in an isolated Google Compute Engine virtual machine that offer a vanilla build environment for every build.
 
-> Container-based infrastructure is currently being [deprecated](https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures). 
-> Please use the [fully-virtualized infrastructure](#fully-virtualized-via-sudo-required) via `sudo: required` instead.
+This has the advantage that no state persists between builds, offering a clean slate and making sure that your tests run in an environment built from scratch.
 
-When specifying `sudo: false`, Travis CI runs each build in a container on a
-shared host via Docker.  The container contents are a pristine copy of the
-Docker image, as guaranteed by Docker itself.
+Your build is routed to this infrastructure automatically, you don't need make any modifications to your `.travis.yml`.
 
-The advantage of running with `sudo: false` is dramatically reduced time between
-commit push to GitHub and the start of a job on Travis CI, which works
-especially well when one's average job duration is under 3 minutes.
+## Container-based infrastructure
 
-*NOTE: The allowed list of packages that may be installed varies between Precise
-and Trusty, with the Precise list being considerably larger at the time of this
-writing.  If the packages you need are not yet available on Trusty, it is
-recommended that you either target `dist: precise` or `sudo: required` depending
-on your needs.*
+> Container-based infrastructure is currently being [deprecated](https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures).
+> Please use the default fully-virtualized [Linux infrastructure](#linux-infrastructure) instead
```

Comment:
```suggestion
> Please remove any `sudo: false` keys in your `.travis.yml` file to use the default fully-virtualized [Linux infrastructure](#linux-infrastructure) instead. 
```

I think this works for the transition but would need to be updated by mid-Dec

What do you think?

Reply:
I was in two minds about leaving the keys in. If we don't there is a single place they are references, in the Overview deprecated infras...